src/common/theme.ts

```diff
@@ -7,7 +7,12 @@ import {
   ThemeOptions,
 } from '@mui/material/styles'
 
-const fontFamily = 'Montserrat'
+import { Montserrat } from '@next/font/google'
+
+export const montserrat = Montserrat({
+  display: 'swap',
+  subsets: ['latin'],
+})
```

Comment:
@AnzheloD I suggest that we also add `cyrillic` to the subsets and switch to `auto` display instead of `swap` as it was jumping between the fonts on local.

```suggestion
export const montserrat = Montserrat({
  display: 'auto',
  subsets: ['latin', 'cyrillic'],
})
```

https://user-images.githubusercontent.com/893608/205339057-ab1a3ece-e3a6-4694-9a22-31c8ddb18159.mp4

Reply:
I had a problem yesterday with display:auto, it was loading only the system fonts, that was the reason to choose swap, but it seems to work fine now

Reply:
I think that happens if the server is not restarted after the font is applied. Thanks for the confirmation.